test_huggingface_live.py

```
@@ -0,0 +1,313 @@
+
+    try:
+        async with huggingface_inference_langchain(config, builder) as client:
+            print(f"✓ LangChain client created: {type(client).__name__}")
+
+            # Test basic invocation
+            from langchain_core.messages import HumanMessage
+            messages = [HumanMessage(content="Say hello in one sentence.")]
+
+            print("\n  Making API call...")
+            result = await client.ainvoke(messages)
+            print(f"✓ API call successful!")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)